AirCastingTests/Logic/SessionsSynchronization/SyncProxyTests/WifiAwareSessionSynchronizerTests.swift

```diff
@@ -0,0 +1,82 @@
+// Created by Lunar on 21/07/2022.
+//
+
+import XCTest
+import Combine
+import Resolver
+@testable import AirCasting
+
+class WifiAwareSessionSynchronizerTests: ACTestCase {
```

Comment:
Awesome! 💯💯💯

Very good tests, already on a pro level. Let's schedule a short call to discuss some further possibilities